.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  kinetic:
+    docker:
+      - image: autonomoustuff/docker-builds:kinetic-perception
```

Comment:
If we want to shave off a couple minutes from the build, we could create an image that already contains all the autoware deps, then rosdep will just double-check and move on.

Reply:
This is true. I had thought about it as well. Would you mind creating a new Dockerfile in https://github.com/astuff/docker-builds in the ros/kinetic/ folder? I'll set up the Docker Hub automated build after that.

Reply:
Will do!

Reply:
@icolwell-as - Any movement on the Docker container?

Reply:
She's a-build-n'. https://cloud.docker.com/u/autonomoustuff/repository/docker/autonomoustuff/docker-builds/timeline